app/services/rss_reader/assembler.rb

```diff
@@ -67,7 +67,7 @@ def parse_and_translate_gist_iframe!(html_doc)
         possible_link = a_tag[0].inner_html
         if /medium\.com\/media\/.+\/href/.match?(possible_link)
           real_link = HTTParty.head(possible_link).request.last_uri.to_s
-          return unless real_link.include?("gist.github.com")
+          break unless real_link.include?("gist.github.com")
```

Comment:
this seems to be a change in behavior though. Before the method returned `nil` and exited from the loop, now the look breaks and the method returns `html_doc`

Reply:
Yeah, you are right. Changed it to `return nil` because otherwise rubocop is at it again.